display_list/display_list_utils.cc

```diff
@@ -578,12 +578,13 @@ bool DisplayListBoundsCalculator::AdjustBoundsForPaint(
   }
 
   if (flags.is_geometric()) {
+    auto sk_path_effect = path_effect_ ? path_effect_->skia_object() : nullptr;
```

Comment:
We should probably add an `SkRect effect_bounds(SkRect)` onto DlPathEffect so we can compute the bounds directly from a DashEffect without having to get Skia involved.